plugins/ja3_fingerprint/ja3_utils.cc

```diff
@@ -0,0 +1,102 @@
+/** @ja3_utils.cc
```

Comment:
I know this is copied from the other ja3_fingerprint.cc header file, but I think it's supposed to be `@file ja3_utils.cc`. Can you fix it in the ja3_fingerprint.cc file too?

Reply:
Could we leave the `ja3_utils.cc` part off completely as in the core, and keep just the `@file` like everywhere else?

Reply:
https://www.doxygen.nl/manual/commands.html#cmdfile

Seems it should be done as you said.